README.md

```diff
@@ -3,3 +3,9 @@ To setup cassandra with docker:
 docker pull cassandra
```

Comment:
why didn't we remove cassandra reference from here

Reply:
my bad :)